boards/arm/waveshare_open103z/pinmux.c

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2020 Stefano Manni
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <device.h>
+#include <init.h>
+#include <drivers/pinmux.h>
+#include <sys/sys_io.h>
+
+#include <pinmux/stm32/pinmux_stm32.h>
+
+/* pin assignments for waveshre_open103z board */
+static const struct pin_config pinconf[] = {
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(usart1), okay)
```

Comment:
On master, we're back adding Kconfig control here:
`#if DT_NODE_HAS_STATUS(DT_NODELABEL(usart1), okay) && CONFIG_SERIAL`
Please update for all devices

Reply:
just force pushed with the required changes